app/ts/simulation/protectors/feeOops.ts

```diff
@@ -2,10 +2,8 @@ import { Simulator } from '../simulator.js'
 import { EthereumUnsignedTransaction } from '../../types/wire-types.js'
 import { SimulationState } from '../../types/visualizer-types.js'
 
-const MAX_FEE = 2000n * 10n**9n
-
-export async function feeOops(transaction: EthereumUnsignedTransaction, _simulator: Simulator, _simulationState: SimulationState) {
-	if (transaction.type === '1559') { if (transaction.maxPriorityFeePerGas < MAX_FEE) return }
-	else if (transaction.gasPrice < MAX_FEE) return
+export async function feeOops(transaction: EthereumUnsignedTransaction, simulator: Simulator, _simulationState: SimulationState) {
+	if (transaction.type === '1559') { if (transaction.maxPriorityFeePerGas < 10n ** 9n) return } // 1.0 nanoEth/gas 
+	else if (transaction.gasPrice < await simulator.ethereum.getGasPrice() * 10n) return // 10 times the estimate gas pice
```

Comment:
```suggestion
	if (transaction.type === '1559' && transaction.maxPriorityFeePerGas < 10n ** 9n) return // 1.0 nanoEth/gas
	if (transaction.type !== '1559' && transaction.gasPrice < await simulator.ethereum.getGasPrice() * 10n) return // 10 times the estimate gas price
```

May be more readable to unnest the first if statements.
This function is supposed to return ``'BIG_FEE' | undefined``?